### PR TITLE
display actioner details

### PR DESCRIPTION
**Pull request by jeberl**
Summary
---------

Add more details to the actioner configs page

Next step, make it editable

Also some small changes that we requested from earlier PR

Test Plan
---------

Screen Shot 2021-04-21 at 3 10 13 PM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,4 @@ pdq/java/bazel-out/
 pdq/java/bazel-testlogs/
 .mypy_cache/
 .vscode/
+hasher-matcher-actioner/.devcontainer/devcontainer.json
```
Comment (eriksendc) [inline .gitignore:20]: I see this is added here... Not sure if we want to do this... If we go this route we'll need a convention to call out when to go get a new copy, and we'll have to store the "official" team copy somewhere outside of source control. I'm pretty sure this should be left in as-is. @schatten what do you think?

File: hasher-matcher-actioner/webapp/src/Settings.jsx [hunk 2]
```diff
@@ -44,44 +46,69 @@ function ActionLabelSettings() {
   return <>Action Label Settings coming here</>;
 }
 
-function WebhookActionerRender() {
-  return <Card>This is a card displaying a WebhookActionerRender</Card>;
+function WebhookActionerRender({url, headers, webhookType}) {
```
Comment (eriksendc) [inline hasher-matcher-actioner/webapp/src/Settings.jsx:49]: This naming is unconventional to me. We see this in the "framework" that was used for SignalOsmosis configs... How about something like `WebhookActioner` or `WebhookActionerDetails`? It's the `Renderer` bit that doesn't feel right.

File: hasher-matcher-actioner/webapp/src/Settings.jsx [hunk 3]
```diff
@@ -44,44 +46,69 @@ function ActionLabelSettings() {
...
+  return (
+    <Card>
+      <h5>WebhookActionerRender</h5>
```
Comment (eriksendc) [inline hasher-matcher-actioner/webapp/src/Settings.jsx:52]: Use `<CardHeader>`, `<CardBody>` and `<CardFooter>`. And if you have a set of attributes, like URL, Webhook Type and Headers, you probably should put a table inside the `<CardBody>`.

Comment (eriksendc) [reply hasher-matcher-actioner/webapp/src/Settings.jsx:52]: https://react-bootstrap.github.io/components/cards

**Review by eriksendc — CHANGES_REQUESTED** [review]
Please revert .gitignore, deal with renaming `WebhookActionerRenderer` and use the card-related React Bootstrap components as mentioned in inline comments.

File: .gitignore (hunk 1 above)
Comment (jeberl) [reply .gitignore:20]: Didnt you need to change this file to set up the dev container? Change dipanjanm name to your unix name?

File: hasher-matcher-actioner/webapp/src/Settings.jsx (hunk 2 above)
Comment (jeberl) [reply hasher-matcher-actioner/webapp/src/Settings.jsx:49]: Good with me

**Review by eriksendc — APPROVED** [review]

**Pull request merged**